packages/node/src/methods/state-channel/deposit/controller.ts

```diff
@@ -34,22 +42,34 @@ export default class DepositController extends NodeController {
     const { store, provider } = requestHandler;
     const { multisigAddress, amount } = params;
 
+    params.tokenAddress = params.tokenAddress
+      ? params.tokenAddress
+      : AddressZero;
+
     const channel = await store.getStateChannel(multisigAddress);
 
     if (
       channel.hasAppInstanceOfKind(
-        requestHandler.networkContext.ETHBalanceRefundApp
+        requestHandler.networkContext.CoinBalanceRefundApp
       )
     ) {
       return Promise.reject(CANNOT_DEPOSIT);
     }
 
     const address = await requestHandler.getSignerAddress();
 
-    const balanceOfSigner = await provider.getBalance(address);
+    if (params.tokenAddress !== AddressZero) {
```

Comment:
Use the variable from my PR. CONVENTION_FOR_ETH_TOKEN_ADDRESS instead of AddressZero. It makes it clear what it means

Reply:
I wasn't waiting for that to be merged in, guess I'll just do it here.